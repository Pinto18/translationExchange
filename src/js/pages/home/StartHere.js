import React from "react";
import { Button } from "semantic-ui-react";
import { Link } from "react-router-dom";


const StartHere = ({text}) => (
					<div className="start-here">
						<Link to="/projects">
							<Button
								icon="right arrow"
								content={text}
								labelPosition="right"
							/>
						</Link>
					</div>

);

export default StartHere;
