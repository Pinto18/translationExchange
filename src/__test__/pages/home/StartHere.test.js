import { shallow, mount, render } from "enzyme";
import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import StartHere from "../../../js/pages/home/StartHere";

describe("Home Component, StartHere", () => {
	const wrapper = shallow(<StartHere />);
	it("is rendered", () => {
		expect(wrapper).toHaveLength(1);
	});
	it("has class, start-here", () => {
		expect(wrapper.hasClass("start-here")).toBe(true);
	});
	it("navigates to project page", () => {
		expect(wrapper.find("Link").props().to).toEqual("/projects");
	});
});
