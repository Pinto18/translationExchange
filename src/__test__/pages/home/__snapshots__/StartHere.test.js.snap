// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component, StartHere has no changes 1`] = `
<div
  className="start-here"
>
  <Link
    replace={false}
    to="/projects"
  >
    <Button
      as="button"
      icon="right arrow"
      labelPosition="right"
      style={
        Object {
          "backgroundColor": "#14D159",
          "boxShadow": "3px 3px 3px rgba(0,0,0,0.6)",
          "color": "white",
          "fontSize": "20px",
          "padding": "1.5vw",
        }
      }
    />
  </Link>
</div>
`;
